WikibaseDataModel.php

```diff
@@ -7,7 +7,10 @@
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
 
-define( 'WIKIBASE_DATAMODEL_VERSION', '5.0.0 alpha' );
+/**
+ * @deprecated since 5.0
```

Comment:
After #595 this constant is of almost no use any more. It doesn't even tell you if the component is loaded or not. By deprecating it I want to document this fact. This does not mean we will remove it any time soon.

If this is considered a blocker for 5.0 we can remove this line.